### ISSUE TITLE
Corrupt XLSX file when drawing image 

### ISSUE THREAD

**Issue by ss-ikeda** [PHPOffice/PhpSpreadsheet#1022]
This is:

```
- [* ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```
I wanted to download xlsx file in which added image file with phpspreadsheet.
Then I am facing issue of excel file broken due to "$drawing" as below source code.

### What is the expected behavior?
No error when open the excel file.

### What is the current behavior?
When opened the xlsx file, it always asked if fix to see. 

### What are the steps to reproduce?
Kindly try below code...

// add code that show the issue here...

use PhpOffice\PhpSpreadsheet\Settings;
use PhpOffice\PhpSpreadsheet\Reader\Xlsx as XlsxReader;
use PhpOffice\PhpSpreadsheet\Writer\Xlsx as XlsxWriter;
use PhpOffice\PhpSpreadsheet\Writer\CSV as CSVWriter;

class TestsController extends AppController{

    public function output(){

        //read template
        $reader = new XlsxReader();
        $spreadsheet = $reader->load(WWW_ROOT.'files/template/empty.xlsx');
        $spreadsheet->setActiveSheetIndex(0);
        $sheet = $spreadsheet->getActiveSheet();

        //add image in excel
        $drawing = new \PhpOffice\PhpSpreadsheet\Worksheet\Drawing();
        $drawing->setName('Logo');
        $drawing->setDescription('Logo');
        $drawing->setPath(WWW_ROOT."docs/C20101Docs/20190404202343.png"); 
        $drawing->setCoordinates('A1');
        $drawing->setWorksheet($sheet);
        unset($drawing);

        //create tmp file
        $tmp_path = WWW_ROOT.'files/test_tmp.xlsx';
        $writer = new XlsxWriter($spreadsheet);
        ob_end_clean();
        $writer->save($tmp_path);

        $content = 'attachment;';
        $content .= 'filename*=UTF-8\'\''.rawurlencode("TEST.xlsx"); 
        $this->response->header('Content-Disposition', $content);
        $this->response->type('application/vnd.openxmlformats-officedocument.spreadsheetml.sheet');
        $this->response->file($tmp_path);

        return $this->response;
    }
}

### Which versions of PhpSpreadsheet and PHP are affected?
PhpSpreadsheet 1.7
PHP 7.0.3
cakephp3.6

**Comment by damijanc** [comment]
just a question, are you missing `drawing->setHeight(36);`?

**Comment by ss-ikeda** [comment]
I tried "drawing->setHeight(36);"
Still facing same issue.

**Comment by dangthanhnhat** [comment]
Try using xls template then output xlsx

**Comment by ss-ikeda** [comment]
I tried xls template then facing this error  "You tried to set a sheet active by the out of bounds index: 0. The actual number of sheets is 0."

**Comment by dangthanhnhat** [comment]
You must use Class Xls
use PhpSpreadsheet\Reader\Xls;

**Comment by ss-ikeda** [comment]
I could download but the output xlsx looks broken...
When opened the xlsx file, it asked if fix to see.

**Comment by dangthanhnhat** [comment]
Did you open template xlsx file and save as xls file or just change the extension ?

**Comment by ss-ikeda** [comment]
I opened template xlsx file and saved as xls file...

**Comment by dangthanhnhat** [comment]
> I opened template xlsx file and saved as xls file...

Try this header
https://phpspreadsheet.readthedocs.io/en/latest/topics/recipes/#http-headers

**Comment by ss-ikeda** [comment]
"$writer->save($tmp_path);" saves same as the output xlsx as temporary file in the server before downloading. I checked the file and tried to open it.
Then confirmed  the issue occurs in the temporary file.

**Comment by beoss** [comment]
I have encountered the same issue but it is only present when loading a template.  If I create a sheet using the "new Spreadsheet()" syntax it works as expected.

**Comment by githubjeka** [comment]
I have same issue too.
https://github.com/PHPOffice/PhpSpreadsheet/issues/877#issuecomment-515455684

**Comment by whosphp** [comment]
> Try using xls template then output xlsx file

This works for me, thank you

**Comment by oleibman** [comment]
FIxed by #1462.
